JpaLayer/pom.xml

```diff
@@ -0,0 +1,154 @@
+<?xml version="1.0"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.axinalis</groupId>
+  <artifactId>JpaLayer</artifactId>
+  <version>1.0</version>
+  <name>JpaLayer</name>
+  <url>http://maven.apache.org</url>
+
+  <properties>
+    <maven.compiler.source>11</maven.compiler.source>
```

Comment:
why not 17? :D

Reply:
Java 11 has lighter images in docker
It's just for my comfort in this project

Reply:
IMO java17 new LTS (be ready use new java features). in Docker you use it as runtime btw.